### DISCUSSION TITLE
When is node 20 available for Git Hub Enterprise Runners?

### DISCUSSION THREAD

**Discussion by ndeeH** [community/community#130592] [Enterprise] [answered]
### Body

all asked in title ;-)

**Comment by queenofcorgis** [comment, marked as answer]
Self-hosted runners will only _automatically_ update to the version advertised by GHES.

Node20 was added to runner version [2.308.0 ](https://github.com/actions/runner/releases/tag/v2.308.0).

You can either either update the runner manually to 2.308+ or, update GHES to a version that advertises 2.308+ which would be [3.11+](https://github.com/github/ghes-actions/blob/enterprise-3.11.12-release/config/runners/runners.json).
